core/src/wallet/bitcoin/api_impl/BitcoinLikeTransactionApi.h

```diff
@@ -109,7 +111,10 @@ namespace ledger {
             std::vector<uint8_t> serializeOutputs() override;
 
             int32_t getVersion() override;
+            
+            void setSignatures(const std::vector<api::BitcoinLikeSignature> & signatures) override;
 
+            void setDERSignatures(const std::vector<std::vector<uint8_t>> & signatures) override;
```

Comment:
Ditto.

Reply:
Comments are present in generated code - Should I report them into class implementation ?

Reply:
Ah, no. My mistake.